local/bin/py/version_getter.py

```diff
@@ -60,11 +61,24 @@ def get_versions(keys):
 Currently only supports synthetics-windows-pl
 '''
 if __name__ == "__main__":
+    github_output = os.getenv('GITHUB_OUTPUT')
     PRODUCTS = ["synthetics-windows-pl"]
     data = get_data()
     keys = get_keys(data)
+    try:
+        current_versions = json.load(open('data/synthetics_worker_versions.json'))
+    except:
+        current_versions = {}
     final_versions = get_versions(keys)
-    print(final_versions)
     
-    with open('data/synthetics_worker_versions.json', 'w') as f:
-        f.write(json.dumps(final_versions, indent=4, sort_keys=True))
+    if current_versions != final_versions:
+        print("New version detected!")
```

Comment:
Is it all right to leave the debugging print statements in the production version?

Reply:
Yep! I kept the echo step in as well, cos I think it helps whoever needs to debug this down the line